docs/adr/013-choice-of-documentation-site.md

```diff
@@ -0,0 +1,71 @@
+# Choice of Documetation Site Framework
+
+Author: Bryce Willey
+
+Date: 2025-06-25
+
+Status: Decided, In progress 
+
+While we have various ways of generating and viewing docs locally (manually
+building javadocs, using [enunciate](008-choice-of-rest-docs.md) to view REST
+endpoint docs), there's no easy way to access this information if you don't
+have a local development environment set up (i.e. new developers or folks who
+don't work on this repo much).
+
+We should host this information on a documentation that is easily findable and
+accessible. The only real choice for where to host it is on GitHub pages, and
```

Comment:
Maybe this was in another doc I haven't read yet - Why is GitHub the only real choice for hosting? Is it the simplest to implement? Requires the least maintenance? Etc.